raincloud/__init__.py

```diff
@@ -77,11 +78,11 @@ def restoreBackup():
         return 
 
     # TODO: get list of services
-    @app.route('/service/list', methods=['GET'])
+    @app.route('/services', methods=['GET'])
```

Comment:
Change the endpoint `service/list` to `services` to better adhere to REST specifications.